test/controllers/declaration/DucrChoiceControllerSpec.scala

```diff
@@ -82,6 +84,18 @@ class DucrChoiceControllerSpec extends ControllerSpec {
       }
     }
 
+    "return 303 (SEE_OTHER)" when {
+      List(STANDARD, CLEARANCE, SIMPLIFIED, OCCASIONAL).foreach { declarationType =>
+        s"journey is $declarationType" in {
```

Comment:
Need to specify that this is testing the situation where the draft dec is an amendment draft